docs/source/install.rst

```diff
@@ -86,39 +86,39 @@ These optional dependencies and their minimum supported versions are listed belo
 +-------------+----------+--------------------------------------------------------------+
 | Dependency  | Version  |                          Description                         |
 +=============+==========+==============================================================+
-|    bokeh    | >=1.0.0  |                Visualizing dask diagnostics                  |
+|     bokeh     | >=1.0.0  |                Visualizing dask diagnostics                  |
```

Comment:
It looks like additional spaces were added which is causing some misalignment in the column width. Could you update to make sure each row has the same width for the "Dependency", "Version", and "Description" columns? Otherwise sphinx won't render the table:

![Screen Shot 2020-03-10 at 12 19 18 PM](https://user-images.githubusercontent.com/11656932/76340514-b43bfc00-62c9-11ea-9494-d3c4a1f4180c.png)

Reply:
Hopefully, it's fixed now.

Reply:
Yep, looks good now. Thanks for updating